lib/rubocop/cop/rails/time_zone.rb

```diff
@@ -51,11 +51,12 @@ def check_time_node(klass, node)
                     !(chain & good_methods).empty?
 
           method_name = (chain & DANGER_METHODS).join('.')
+          safe_method_name = safe_method(method_name, node)
 
           add_offense(node, :selector,
                       format(MSG,
                              "#{klass}.#{method_name}",
-                             "#Time.zone.#{method_name}")
+                             "#Time.zone.#{safe_method_name}")
```

Comment:
There should not be `#` before `Time` and the code snippets should surrounded in backticks, so they're properly coloured in the terminal.